doc/classes/Particles2D.xml

```diff
@@ -305,26 +305,27 @@
 			Number of horizontal frames in [code]texture[/code].
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
-			Amount of time each particle will exist. Default value: [code]1[/code].
+			Amount of seconds each particle will exist. Default value: [code]1[/code].
 		</member>
 		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
 			If [code]true[/code] particles use the parent node's coordinate space. If [code]false[/code] they use global coordinates. Default value: [code]true[/code].
 		</member>
 		<member name="normal_map" type="Texture" setter="set_normal_map" getter="get_normal_map">
 		</member>
 		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot">
-			If [code]true[/code] only [code]amount[/code] particles will be emitted. Default value: [code]false[/code].
+			If [code]true[/code] does only one emission cycle or if set [true] while one is ongoing, doesn't start the next one. Default value: [code]false[/code].
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time">
+			Particle system starts as if it had already ran for this many seconds.
```

Comment:
>had already ran

`had already run` or `had already been running`

Reply:
Yes, of course, thanks, fixed.